src/components/Award.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from "react";
+import { AwardPeriod } from "../../generated/definitions/AwardPeriod";
+import { useTranslation } from "react-i18next";
+import { format, parseISO } from "date-fns";
+import viewIcon from "../assets/view.svg";
+import classNames from "classnames";
+import "./Award.css";
+
+type AwardProps = {
+  el: AwardPeriod;
+  index: number;
+  key?: number;
+  popModal: (el: AwardPeriod) => void;
+};
+
+export const Award: React.FunctionComponent<AwardProps> = props => {
```

Comment:
I suggest to use destructuring when taking props ie: 

```suggestion
export const Award: React.FunctionComponent<AwardProps> =({ el, key, popModal  }) => {
```

in this way you

1.  avoid to reference `props.<property>` every time
2. get an hint from the IDE on which props is used / unused